awx/main/notifications/hipchat_backend.py

```diff
@@ -38,7 +38,7 @@ def send_messages(self, messages):
                 r = requests.post("{}/v2/room/{}/notification".format(self.api_url, rcp),
                                   params={"auth_token": self.token},
                                   verify=False,
-                                  json={"color": self.color,
+                                  json={"color": self.color.lower(),
```

Comment:
I think my only concern here is if `self.color == None` which is not super possible in our current notification system but could find its way to that. So could do something like

`self.color.lower() if self.color is not None else None` or some such thing.